app/services/events/EventsParsingService.scala

```diff
@@ -97,4 +69,46 @@ trait EventsParsingService {
       }
     }
   }
+
+  lazy val df = DateTimeFormat.forPattern("HH:mm")
+
+  private def event(items: String): Event = {
+    event(items.split(", ?", -1))
+  }
+
+  private def event(items: Array[String]): Event = {
+    val eventType = EventType.withName(items.head.replaceAll("\\s|-", "_").toUpperCase)
+    val eventDescription = items(1)
+    val location = items(2)
+    val venue = VenueType.withName(items(3).replaceAll("\\s|-", "_").toUpperCase)
+    val date = LocalDate.parse(items(4), DateTimeFormat.forPattern("dd/MM/yy"))
+    val startTime = df.parseLocalTime(items(5))
+    val endTime = df.parseLocalTime(items(6))
+    val capacity = items(7).toInt
+    val minViableAttendees = items(8).toInt
+    val attendeeSafetyMargin = items(9).toInt
+
+    if(eventDescription.length > 10) throw new Exception("Event description cannot be more than 10 characters")
```

Comment:
This restriction is not reported to a user so perhaps we want to get a `slice()` here? Check with Alastair

I think it'll be pretty annoying if we have 30 events and 25 of them have description > 10chars. Also, 10 chars seems a bit short for a description. Check with Alastair